File: OmkarFCUBEAPP/src/app/models/gstsalesregisterrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Gstsalesregisterrptmodel } from "./gstsalesregisterrptmodel";

export class Gstsalesrptlistmodel {
    gstSalesList: Gstsalesregisterrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/FinTrans/Models/GstSalesRegisterRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FinTrans.Models
{
    public class GstSalesRegisterRptListModel
    {
        public List<GstSalesRegisterRptModel> GstSalesList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
